Src/Support/GoogleApis.Core/Portable/CompatExtensions.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using System.Text;
+
+namespace Google
+{
+    internal static class CompatExtensions
```

Comment:
I'd personally create a Compatibility namespace and a ReflectionExtensions type within it.